### PR TITLE
Add pdf output

### PR DESCRIPTION
**Pull request by chrisburr**
Could be useful so teachers can easily print the lessons. Needs some changes to fix the css in lhcb/gitbook-plugin-panels#2 but otherwise works.

[Example output](https://www.dropbox.com/s/6dk9ud93dlzezis/analysis-essentials.pdf?dl=1)

### CODE REVIEW

**Comment by alexpearce** [comment]
Nice addition!

Maybe the `PATH` setting [needs to be in `before_script`](https://docs.travis-ci.com/user/installing-dependencies/#Installing-Projects-from-Source) rather than `install`? I don't know if the environment between `install` and `script` is shared 🤔

**Comment by alexpearce** [comment]
Maybe can just add [`calibre` and `calibre-bin`](https://github.com/travis-ci/apt-package-whitelist/blob/9f2c1589efa05375f74f9c22c98aea9822b5af5d/ubuntu-trusty#L403-L404) to the [list of packages to install](https://docs.travis-ci.com/user/installing-dependencies/#Adding-APT-Packages).

**Comment by chrisburr** [comment]
I think this is working now! 👍

File: Makefile [hunk 1]
```diff
@@ -14,8 +18,9 @@ test: all
 
 clean:
 	@rm -rf _book
+	@rm -f book.pdf analysis-essentials.pdf
 	@rm -rf node_modules
 	@rm -f tests/*.pyc
 
-publish-travis: all
+publish-travis: all pdf
 	@ghp-import -n ./_book && git push -fq https://${GH_TOKEN}@github.com/$(TRAVIS_REPO_SLUG).git gh-pages > /dev/null
```
Comment (alexpearce) [inline Makefile:26]: I'm not super familiar with it, but I think `ghp-import` will replace the `gh-pages` branch with the contents of the `_book` directory. The PDF made by `make pdf` is `analysis-essentials.pdf` in the _root_ of the repo, so we should `cp` it to `_book` in this rule.

**Review by alexpearce — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (chrisburr) [reply Makefile:26]: I *think* it gets copied by the `otherlink` plugin.

Comment (alexpearce) [reply Makefile:26]: OK, let's merge then, and if you're wrong you get the privilege of fixing it 😉

**Review by alexpearce — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (chrisburr) [reply Makefile:26]: [It works!!!!](https://lhcb.github.io/analysis-essentials/analysis-essentials.pdf) 🎉

Comment (alexpearce) [reply Makefile:26]: 😵

**Pull request merged**